src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -670,10 +672,51 @@ public void LogTelemetry(string eventName, IDictionary<string, string> propertie
         #endregion
 
         #region IBuildEngine7 Members
+
         /// <summary>
         /// Enables or disables emitting a default error when a task fails without logging errors
         /// </summary>
         public bool AllowFailureWithoutError { get; set; } = false;
+
+        #endregion
+
+        #region IBuildEngine8 Members
+
+        int runningTotal = 0;
```

Comment:
```suggestion
        private int runningTotal = 0;
```
? Also, I assume you put this here because it has to do directly with Request/ReleaseCores, but do you think that merits a comment?